app/src/main/java/http_server/ResponseHandler.java

```diff
@@ -3,37 +3,41 @@
 public class ResponseHandler {
   Response response = new Response();
 
-  public Response buildSimpleGetWithBodyResponse() {
-    response.setResponseBody("Hello world");
+  public Response buildResponse( String status, String responseBody) {
```

Comment:
As you know, I like this refactor we did here to just use one method :)

Reply:
Yes! this was a good refactor :)